.github/dependabot.yml

```
@@ -17,57 +17,13 @@ updates:
   - kind/skip-release-notes
   - component/github-actions
 - package-ecosystem: "gomod"
-  directory: "/"
+  directories:
+    - "/bindings/go/*"
```

Do we have to update this field, when we add go-code to other locations?

---

```suggestion
      - "**/*"
```
This should do the job as well and prevent that we have to update the config each time we add go-code to another location

---

> i dont want to update go code everywhere right?

My assumption is that `dependabot` updates code-dependencies. So, every go-code that we write and that has dependencies should be regularly checked by `dependabot`. Did I misunderstand something?

As far as I understood this repository, `binding/go` will contain the library code. But I expected a cli-tool that is also written in go and is located in `cli/`. Maybe this assumption was wrong. If my assumption was correct, then we need to adjust the `dependabot` configfile, when we introduce new go-code.

---

Looked at this again, I think youre right. However I do not really want dependabot to have to scan every single directory in our repository